integration-tests/utils/__init__.py

```diff
@@ -0,0 +1,9 @@
+import sh
+
+
+def rhcd_service_is_active():
```

Comment:
The doc string is missing here. BTW: the term `rhcd` could be misleading, because upstream name is `yggdrasil`.